src/dojos/cd-find-dojo.vue

```diff
@@ -232,7 +228,7 @@
 
             &--reduced {
               min-width: 310px;
```

Comment:
Is there a difference between having (max-width == min-width || width)?

Reply:
Not sure what you mean by this

Reply:
you set min & max width on the same class.
is there a reason to not use "width" instead?

Reply:
width was not working as we wanted it to on it's own

Reply:
Updated this